core/runtime/snapshot.rs

```diff
@@ -110,22 +110,73 @@ impl SnapshotStoreDataStore {
   }
 }
 
+/// Options for [`create_snapshot`].
+///
+/// See example use in [this blog post][1].
+///
+/// [1]: https://deno.com/blog/roll-your-own-javascript-runtime-pt3#diving-into-createsnapshotoptions
```

Comment:
I'd rather not link to the blog post here as it will only drift apart as we iterate on the API

Reply:
Fair. 

 - [ ] Now that there's an example that will be regularly built, I can link to that instead.